itest/lnd_psbt_test.go

```diff
@@ -1919,3 +1921,104 @@ func testPsbtChanFundingWithUnstableUtxos(ht *lntest.HarnessTest) {
 	block = ht.MineBlocksAndAssertNumTxes(1, 1)[0]
 	ht.AssertTxInBlock(block, txHash)
 }
+
+// lnrpcOutpointToStr returns a string representation of an lnrpc.OutPoint.
+func lnrpcOutpointToStr(outpoint *lnrpc.OutPoint) string {
```

Comment:
this seems to be place in a helper I could imagine a lot of test might use this function.